stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -417,6 +417,20 @@ extension StringProtocol where Index == String.Index {
     return self._ephemeralString as NSString
   }
 
+  func _withCore<Result>(_ apply: (_StringCore) throws -> Result) rethrows -> Result {
```

Comment:
It was a convenient way to hack them in without causing code explosions.  But yes, it's probably a mistake.